src/content/code/plugin-api/layers/findall-layer.ts

```
@@ -0,0 +1,27 @@
+// @ts-nocheck
+
+//1. Define a search function to find all layers with a specific visibility setting
+const searchVisibleLayers = (layer) => layer.isVisible;
+
+// Use the findAll method to get all visible layers
+const visibleLayers = reearth.layers.findAll(searchVisibleLayers);
+
+// Output the IDs of the found layers
+console.log(
+  "Visible layers found:",
+  visibleLayers.map((layer) => layer.id)
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)